h/static/scripts/session.js

```diff
@@ -156,7 +156,7 @@ function session($http, $resource, $rootScope, flash, raven, settings) {
 
   function process(data, headersGetter, status) {
     if (status < 200 || status >= 500) {
-      return;
+      return data;
```

Comment:
This is again a change from current behaviour, and working out all the implications is not trivial. `process` is used as a response transformer, which affects the behaviour of `resource`, which is used throughout this module.

Are you sure this is a safe change?